src/diagrams/sequence/sequenceRenderer.js

```diff
@@ -168,7 +168,7 @@ export const bounds = {
 
 const _drawLongText = (text, x, y, g, width) => {
   let textHeight = 0;
-  const lines = text.split(/<br\/?>/gi);
+  const lines = text.split(/<br ?\/?>/gi);
```

Comment:
It think what's needed here, to follow [the spec](http://www.w3.org/TR/REC-xml/#sec-starttags) isn't `<br ?\/?>` but instead `<br\s*\/?>` as HTML allows an arbitrary amount of whitespace (as opposed to 0 or 1 characters of whitespace) between the end of an HTML tag and that whitespace can be space tab carriage return or line feed.

Reply:
I'll see if I can PR that minor tweak today

Reply:
There's more places than this in mermaid where regex for line breaks is used. These differ in their abilities to find certain line break notations. Maybe this should be a new issue to get this done for all mermaid diagram types.

Reply:
Good point, I'll take a look and open an issue (or a PR if it's easily solvable)